.docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM selenium/standalone-chrome
+
+# add mintapi to path
+ENV PATH=$HOME/.local/bin:$PATH
+
+RUN echo "**** install packages ****" && \
+    sudo apt-get update && \
+    sudo apt-get install -y python3-pip && \
+    pip3 install mintapi && \
+    echo "**** cleanup ****" && \
```

Comment:
I'd personally like it better if the "cleanup" was a different job of the Docker file.  That way we can differentiate between errors in the install job and the cleanup job.  They're also separate responsibilities of the container.

Reply:
That would make the layer larger however. It is standard practice when creating docker files for each layer to clean up itself so layers are more independent.